components/browser/engine-gecko/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngineSession.kt

```diff
@@ -1158,3 +1216,22 @@ internal fun EngineSession.LoadUrlFlags.getGeckoFlags(): Int = if (contains(ALLO
 } else {
     value
 }
+
+/**
+ * Create a Content Disposition formatted string with the receiver used as the filename and
+ * file extension set as PDF.
+ */
+internal fun String.pdfContentDisposition() = take(MAX_TITLE_LENGTH).run {
+    "attachment; filename=$this.pdf;"
+}
```

Comment:
Maybe this should be on `DownloadUtils`, wdt?

Reply:
Can do. It's very specific to the GeckoSession implementation which is why it's here, but maybe there is value in it being in a generic component?

Reply:
Having it on a generic component give the opportunity for re-usability, and discoverability as most related code it's already `DownloadUtils` :)